### ISSUE TITLE
Bedrock: support for custom endpoints

### ISSUE THREAD

**Issue by ladrians** [langchain-ai/langchainjs#2376]
Is there a change to add extra configuration to define a custom endpoint to test Bedrock?

I was provided an URL similar to the following one:

```
https://{instance}.execute-api.{region}.amazonaws.com/{env}/v1/chat/completions
```

but noticed it goes internally to a fixed one, so I receive this kind of error

```
Failed to access underlying url 'https://bedrock.us-east-1.amazonaws.com/model/anthropic.claude-v2/invoke
```

thanks

**Comment by jacoblee93** [comment]
This would be lovely - would welcome a PR or maybe @Benyuel or @vitaly-ps would be up for a crack at it?

**Comment by Benyuel** [comment]
@ladrians, is that a valid bedrock url?  I'm not seeing it in the documentation: https://docs.aws.amazon.com/bedrock/latest/APIReference/API_Operations.html

Also, currently the bedrock LLM is tightly coupled to the [InvokeModelWithResponseStream](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_InvokeModelWithResponseStream.html) endpoint and it's response format with streaming, so I'm not sure if we should expose/allow other urls (if there are some).

**Comment by ladrians** [comment]
Yes it is, we have access to the following models referencing the initial url mentioned, 'anthropic.claude-v1', 'anthropic.claude-v2', 'anthropic.claude-instant-v1', 'amazon.titan-tg1-large', 'ai21.j2-ultra', 'ai21.j2-mid'.

**Comment by jacoblee93** [comment]
Fixed by #2614 and will be live in the next release (0.0.149)!
